bootstrap.sh

```diff
@@ -1,24 +1,25 @@
 #!/usr/bin/env bash
 # Bootstrap script for devxp-linux. Run this the first time you set up your WSL system.
 
-# Upgrade
-sudo apt update
+# Upgrade the distro.
+sudo apt-get update
```

Comment:
isn't `apt` supposed to be a "substitute" for `apt-get`? Any reason we want to use `apt-get` explicitly ?